src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/JavaNCSSCheck.java

```diff
@@ -83,7 +92,73 @@
  *   &lt;property name="methodMaximum" value="40"/&gt;
  * &lt;/module&gt;
  * </pre>
+ * <p>Example:</p>
+ * <pre>
+ * public void test() {
+ *   System.out.println("Line 1");
+ *   // another 38 lines of code
+ *   System.out.println("Line 40") // OK
+ *   System.out.println("Line 41") // violation, the method crosses 40 lines of uncommented code
```

Comment:
it is not `uncommented code`, it is `non commented lines`

Reply:
Done